### PR TITLE
[stable/filebeat] Creation of a PodSecurityPolicy for Filebeat

### PR DESCRIPTION
**Pull request by marc-sensenich**
Signed-off-by: Marc Sensenich <sensenichm91@gmail.com>

#### What this PR does / why we need it:
Allows for Filebeat to run in a cluster with the PodSecurityPolicy admission controller enabled

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by marc-sensenich** [comment]
There are 2 open PRs that may need to be coordinated across
- https://github.com/helm/charts/pull/8889 for the privileged usage for OpenShift
- #9143 for the bumping to the latest release of the Elastic Beats

**Comment by marc-sensenich** [comment]
@paulczar would you have a chance to review?

File: stable/filebeat/README.md [hunk 1]
```diff
@@ -56,7 +56,9 @@ The following table lists the configurable parameters of the filebeat chart and
 | `affinity`                                               |                                                                                                          | `{}`                                               |
 | `rbac.create`                                            | Specifies whether RBAC resources should be created                                                       | `true`                                             |
 | `serviceAccount.create`                                  | Specifies whether a ServiceAccount should be created                                                     | `true`                                             |
-| `serviceAccount.name`                                    | he name of the ServiceAccount to use                                                                     | `""`                                               |
+| `serviceAccount.name`                                    | the name of the ServiceAccount to use                                                                     | `""`                                               |
```
Comment (paulczar) [inline stable/filebeat/README.md:59]: `podSecurityPolicy.enabled` not documented.  looks like the `podSecurityPolicy.name` is misdocumented with `enabled` description ?

**Review by paulczar — CHANGES_REQUESTED** [review]
just the one documentation nitpick.  otherwise I think this looks fine.

File: stable/filebeat/README.md (hunk 1 above)
Comment (marc-sensenich) [reply stable/filebeat/README.md:59]: Thanks for the catch, `name` is not needed as it uses the fullname from the helpers. This has been fixed in https://github.com/helm/charts/pull/9151/commits/d73a3a217feecce864a6c593d652f56c0b21cf39

**Pull request merged**